runtime/browser/vr-demo/demo-context-factory.js

```diff
@@ -0,0 +1,40 @@
+/**
```

Comment:
As I mentioned, I wasn't sure about the fate of demo-context-factory. If you remove the view setup, there isn't much boilerplate being encapsulated and it's no longer useful for unifying demo-flow test from the browser demo.

Normally I prefer to be less sloppy, but I figured I would just leave it as-is for a short time, rather than further delay this already over-large PR.